web/src/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
```

Comment:
i'd recommend to changing this to a css module instead of using global css

Reply:
oh yea iI haven't changed anything yet within the app. Going to setup the pipeline first before going there.

Reply:
updated